compiler/mio-tflite280/CMakeLists.txt

```diff
@@ -44,9 +44,16 @@ if(NOT TensorFlowGEMMLowpSource_FOUND)
   return()
 endif(NOT TensorFlowGEMMLowpSource_FOUND)
 
+nnas_find_package(TensorFlowRuySource EXACT 2.8.0 QUIET)
+
+if(NOT TensorFlowRuySource_FOUND)
+  return()
+endif(NOT TensorFlowRuySource_FOUND)
+
 add_library(mio_tflite280_inc INTERFACE)
 target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowSource_DIR}")
 target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowGEMMLowpSource_DIR}")
+target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowRuySource_DIR}")
```

Comment:
Could you please explain why you added RUY header includes to `mio-tflite` ?
I could guess but this module is for IO of tflite files not kernels.

Reply:
I found that `FoldDepthwiseConv2DPass` in `luci_pass` is using `mio-tflite` which includes `GEMMLowpSource`.

Similarly, I thought that `FoldFullyConnectedPass` can use `mio-tflite` which includes `Ruy`.

Reply:
I'm not sure but I think GEMMLowpSource was added from dependency to use tflite source codes.

> Similarly, I thought that FoldFullyConnectedPass can use mio-tflite which includes Ruy.

Yes, logically this is OK.
The thing that makes me hesitate is that `mio-tflite` is getting complicated.
I'll post an issue to separate things.